src/azure-cli-core/azure/cli/core/commands/arm.py

```diff
@@ -20,7 +20,7 @@
 
 logger = azlogging.get_az_logger(__name__)
 
-regex = re.compile('/subscriptions/(?P<subscription>[^/]*)/resourceGroups/(?P<resource_group>[^/]*)'
+regex = re.compile('/subscriptions/(?P<subscription>[^/]*)/resource[gG]roups/(?P<resource_group>[^/]*)'
```

Comment:
Question: Should we have full case-insensitive regex instead just `resourceGroup` with g / G?